src/components/Picklist-item.vue

```diff
@@ -31,6 +32,9 @@ export default defineComponent({
           this.router.push({ path: `/picklist-details/${picklist.picklistId}` });
         }
       });
+    },
+    getTime(time: any) {
```

Comment:
What's the point of using typescript if types are not used?

```suggestion
    getTime(time: number) {
```
https://moment.github.io/luxon/api-docs/index.html#datetimefrommillis

Reply:
@ps011 Updated, thank you for the review.